nautobot/core/settings.py

```diff
@@ -868,8 +872,9 @@
     "Banners": ["BANNER_LOGIN", "BANNER_TOP", "BANNER_BOTTOM"],
     "Change Logging": ["CHANGELOG_RETENTION"],
     "Device Connectivity": ["NETWORK_DRIVERS", "PREFER_IPV4"],
+    "Device Constraints": ["DEVICE_UNIQUENESS"],
```

Comment:
I'd probably just keep this in the `Natural Keys` section rather than creating a new section for this one item. WDYT?